drv/user-leds/src/main.rs

```diff
@@ -144,11 +143,10 @@ impl idl::InOrderUserLedsImpl for ServerImpl {
         index: usize,
     ) -> Result<(), RequestError<LedError>> {
         let led = Led::from_usize(index).ok_or(LedError::NotPresent)?;
-        let already_blinking = self.blinking.values().any(|b| *b);
+        let any_blinking = self.blinking.values().any(|b| *b);
         self.blinking[led] = true;
-        if !already_blinking {
-            led_on(led);
-            self.blink_state = false;
+
+        if !any_blinking {
```

Comment:
I'd suggest keeping the `led_on(led)` call here, so that going from off to blinking is obviously right away (otherwise, you have to wait for a `BLINK_INTERVAL`).

Reply:
So, in practice, if I leave that line in, the LED starts off. I'm not sure why. Was it blinking immediately before?

Reply:
That confuses me!  The point of this line was to get immediate feedback when calling `led_blink` (if other LEDs weren't already blinking), and I'm pretty sure it worked.  I'd have to think through it a little more to understand why it's not working in this case.

Reply:
Yeah, I assumed it had something to do with the previously ambiguous nature of `blink_state` (where `blink_state` of `false` appeared to mean on?). But, had to remove this to make the feedback immediate.